app/Controllers/Timetable.hs

```diff
@@ -20,7 +20,6 @@ import Export.LatexGenerator
 import Export.PdfGenerator
 import Happstack.Server
 import MasterTemplate
-import Response.Image (returnImageData)
```

Comment:
This change is appropriate, but please also modify the `Util.Helpers` import to explicitly name the functions being imported. This is a good practice to do in general, so I'm comfortable doing that here.